### PR TITLE
[Z2M] Support zigbee network adapters

### PR DESCRIPTION
**Pull request by moodyblue**
Adds network coordinator support to zigbee2mqtt install. Fixes https://github.com/openhab/openhabian/issues/1868

Test methodology:

1) Install Mosquitto
2) Plug zigbee usb dongle in usb2 port
3) Install Z2M: user is prompted to choose either usb or network installation. Choose usb
4) Verify configuration.yaml and connect to openhabian:8081 to confirm that Z2M is running
5) Uninstall Z2M
6) Install Z2M: user is prompted to choose either usb or network installation. Choose network
7) Verify configuration.yaml and connect to openhabian:8081 to confirm that Z2M is running

Signed-off-by: Hugo Castelo Branco [hugo.castelobranco@gmail.com](mailto:hugo.castelobranco@gmail.com) according to Developer's Certificate of Origin 1.1

### CODE REVIEW

**Comment by moodyblue** [comment]
Will do some changes to the code. Closing temporarily.

**Comment by moodyblue** [comment]
Please review. I think that now it contains your instructions.

File: includes/zigbee2mqtt/configuration.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ mqtt:
   user: "%user%"
   password: "%password%"
 serial:
-  port: "/dev/serial/%adapter"
```
Comment (mstormi) [inline includes/zigbee2mqtt/configuration.yaml:11]: is %adapter correct or is the % at the end missing ? The other placeholders have it.

(note reviewing code is not about testing if your solution works)

**Review by mstormi — CHANGES_REQUESTED** [review]

File: includes/zigbee2mqtt/configuration.yaml (hunk 1 above)
Comment (moodyblue) [reply includes/zigbee2mqtt/configuration.yaml:11]: The missing % at the end comes from the original code. I will test if adding a leading % makes any difference.

**Pull request merged**